apps/meteor/ee/server/local-services/omnichannel/OmniJobSchedularService.ts

```diff
@@ -0,0 +1,93 @@
+import type { Job } from '@rocket.chat/agenda';
+import { Agenda } from '@rocket.chat/agenda';
+import { MongoInternals } from 'meteor/mongo';
+import type { Db } from 'mongodb';
+
+import { ServiceClassInternal } from '../../../../server/sdk/types/ServiceClass';
+import { schedulerLogger } from '../../../app/livechat-enterprise/server/lib/logger';
+import { OmniEEService } from '../../sdk';
+import type { IOmniJobSchedularService } from '../../sdk/types/IOmniJobSchedularService';
+
+const SCHEDULER_NAME = 'omnichannel_scheduler';
+
+export enum OMNI_JOB_NAME {
+	AUTO_CLOSE_ON_HOLD_CHAT = 'auto-close-on-hold-chat',
+	AUTO_TRANSFER_UNANSWERED_CHAT = 'auto-transfer-unanswered-chat',
+}
+
+export class OmniJobSchedularService extends ServiceClassInternal implements IOmniJobSchedularService {
+	protected name = 'omni-job-schedular';
+
+	scheduler: Agenda;
+
+	logger = schedulerLogger;
+
+	db: Db;
+
+	constructor() {
+		super();
+
+		const { db } = MongoInternals.defaultRemoteCollectionDriver().mongo;
```

Comment:
A service can receive the DB as param 👀 (like, Authorization service iirc)

Reply:
Done :+1: